src/Microsoft.AspNetCore.Hosting.Abstractions/IWebHostBuilder.cs

```diff
@@ -75,6 +91,6 @@ public interface IWebHostBuilder
         /// </summary>
         /// <param name="configureDelegate">The delegate for configuring the <see cref="IConfigurationBuilder" /> that will be used to construct an <see cref="IConfiguration" />.</param>
         /// <returns>The <see cref="IWebHostBuilder"/>.</returns>
-        IWebHostBuilder ConfigureConfiguration(Action<WebHostBuilderContext, IConfigurationBuilder> configureDelegate);
+        IWebHostBuilder ConfigureAppConfiguration(Action<WebHostBuilderContext, IConfigurationBuilder> configureDelegate);
```

Comment:
@DamianEdwards What do you think about this method? We wanted to keep the Configure part since it matches all the existing methods and is nice in intellisense, and using something other than Configuration as the description part seemed wrong. So breaking it up with a word so that it doesn't seem so circular was our best suggestion so far.